Svc/Framer/docs/sdd.md

```diff
@@ -20,6 +20,12 @@ in each frame; typically it is a frame header, a data packet, and a hash value.
 You can use the standard F Prime downlink protocol implementation.
 This implementation works with the F Prime ground data system (GDS).
 
+`Svc::Framer` is designed to act alongside instances of the
+[communication adapter interface](https://nasa.github.io/fprime/Design/communications-adapter-interface.html) and thus
+forwards statuses from any downstream communication adapter. In order to not violate this interface protocol,
+`Svc::Framer` will emit a success status anytime a buffer is received but no framed packet is sent (e.g. in the case of
```

Comment:
I think this should say "any time a buffer is received, regardless of whether a corresponding framed packet is sent". The current wording implies that a status is sent if and only if a buffer is received and no framed packet is sent.

Reply:
We may also want to say that we always return success here because any downstream failures are handled downstream.

I'm also a little confused as to why the communication adapter protocol requires us to emit a status if the status is always success. Maybe it's to preserve the option of hooking up the ComQueue to something that could return a failure status? We may also want to say that.